bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/CPU.java

```diff
@@ -1915,6 +1931,9 @@ private static void execTypeCastOpcodes(WorkerExecutionContext ctx, WorkerData s
             case InstructionCodes.ANY2DT:
                 handleAnyToRefTypeCast(ctx, sf, operands, BTypes.typeTable);
                 break;
+            case InstructionCodes.ANYSTM:
```

Comment:
Shall we rename this to ANY2STM, just to stay with the convention?

Reply:
@SupunS will incorporate this into next PR. Thanks.